### ISSUE TITLE
Back end not responding in production

### ISSUE THREAD

**Issue by russtuck** [gordon-cs/gordon-360-ui#622]
People search in production, https://360.gordon.edu/people, only has the "all" option for hall, major, minor, state, country, department, and building.  (It does have proper options for class.) 
This breaks almost all its "advanced search" features.  

360train does not have this problem.  

I don't know if this is a front-end or back-end problem, but I'm putting it here since it's visible in the UI.

This is critical to fix, and I hope it's an easy fix.

**Comment by NathanielRudenberg** [comment]
I can take a quick look at it this evening. I think it's a back-end problem that we've seen before.
________________________________

**Comment by russtuck** [comment]
This seems to be just one symptom of a larger problem with the back-end not responding, hence the new name.  More info from email thread...

Chris Hansen wrote:
I just spoke with Chris Carlson about this, and he had confirmed the issue with other users.  I was getting this at logon:

![image001](https://user-images.githubusercontent.com/12618649/62903986-93f78f00-bd32-11e9-801b-f9c64c9c9f1e.png)
It was working fine for my session that was already logged in, but I couldn’t re-authenticate after logging out.

Rebooting the server didn't help.

**Comment by NathanielRudenberg** [comment]
The problem that I thought was the cause does not seem to be an issue, which leads me to agree with you that it is something bigger.

I don't know whether anything was changed on the API itself when it stopped responding. @chris-hansen is there a chance it could be a config issue on the network?

**Comment by chris-hansen** [comment]
Ronnie and I did several network Wireshark captures yesterday to see if anything was being blocked, but the traffic seemed to be flowing properly.  We also temporarily disabled all of the involved firewalls, but that didn't make any changes, so we turned the firewall back on.  One interesting note that is that the RCI site is being served on the same server, and using very similar authentication logic, but it is still working at the moment.  Jay and I are working at the moment to compare the differences in the logic between the two sites.

**Comment by russtuck** [comment]
Chris and Jay pursued this heroically, and found it was caused by a database change for reasons unrelated to 360.  They fixed the problem!

We also added monitoring for the backend, so we'll know as soon as a problem like this starts again.  Since the backend was running but serving 500s, this is only possible (with current NodePing monitoring) because of the addition of guest mode last summer, which provides API calls that don't require authentication.
